src/pages/patterns/global-header/index.mdx

```diff
@@ -17,38 +17,41 @@ Users rely on the global header to navigate and orient themselves in your UI. Th
   <AnchorLink>Accessibility</AnchorLink>
   <AnchorLink>Related components</AnchorLink>
   <AnchorLink>References</AnchorLink>
+
```

Comment:
We decided to keep the "Feedback" anchor link for all production patterns. I'd add it in here as a suggestion but I don't think I'll get the indentation right.